sum.py

```diff
@@ -13,6 +13,7 @@ def my_sum(a, b):
     """
     assert (is_number(a) and is_number(b)), 'Both addends should be numbers'
     return a - b
+    return a + b
```

Comment:
This adds one extra line but leaves the original one intact. So the resulting code is not valid.